Automattic/jetpack
# Connection UI: init Admin class just once

#### Changes proposed in this Pull Request:

* Make sure we only initialize the main `connection-ui` Admin class once since it may be called multiple times (e.g. in the Jetpack & Backup plugins).

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No.

#### Testing instructions:

* With latest master pulled down, have the Jetpack plugin active, and the Jetpack Backup plugin built and active.
* You will notice two submenu items for "Connection Manager" under the wp-admin Tools menu, before:

![before](https://user-images.githubusercontent.com/15803018/117886615-b5b9d900-b26c-11eb-886d-867f06d0c12e.png)

* After you pull down the changes in this branch, the Connection Manager should only show up once, after:

![after](https://user-images.githubusercontent.com/15803018/117886964-3678d500-b26d-11eb-846d-ff5f2a8c7155.png)
